cfgov/jinja2/v1/owning-a-home/mortgage-closing/index.html

```diff
@@ -0,0 +1,162 @@
+{% extends "_layouts/oah-base.html" %}
+{% import "molecules/social-media.html" as social_media with context %}
+{% set active_page = "mortgage-closing" %}
+{% block title %}Questions about your closing?{% endblock title %}
+{% block metadescription %}Are you about to close on your mortgage loan? The CFPB's checklist and interactive sample Closing Disclosure form will help you close with confidence.{% endblock metadescription %}
+
+{% block content %}
+<main class="content content__2-1 content__bleedbar form-resources" id="main" role="main">
+  <section class="hero hero__mortage-closing">
```

Comment:
No, they are still using an older style here that we don't support in CF4 so I left these in place.